internal/driver/webui.go

```diff
@@ -210,6 +210,9 @@ func defaultWebServer(args *plugin.HTTPServerArgs) error {
 	mux := http.NewServeMux()
 	mux.Handle("/ui/", http.StripPrefix("/ui", handler))
 	mux.Handle("/", redirectWithQuery("/ui", http.StatusTemporaryRedirect))
+	// Enable self-profiling handlers if net/http/pprof is imported by the main
+	// program.
```

Comment:
Are there going to be users who are unhappy about having this access point opened?